pkgs/top-level/all-packages.nix

```diff
@@ -1532,6 +1532,8 @@ with pkgs;
 
   davinci-resolve = callPackage ../applications/video/davinci-resolve { };
 
+  dmodman = callPackage ../tools/games/dmodman { };
+
```

Comment:
```suggestion
```
New top-level packages should go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md